sources/i18n/fr_FR/localization.po

```diff
@@ -0,0 +1,171 @@
+"PO-Revision-Date: 2015-06-14 22:37+0200\n"
+"Last-Translator: root <root@ustelo>\n"
+"Language-Team: French\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: views/layout.html.php:27
+msgid "Wifi Hotspot"
+msgstr "Point d'accès WiFi"
+
+#: views/layout.html.php:49
+msgid "Error"
+msgstr "Erreur"
+
+#: views/layout.html.php:54 views/settings.html.php:122
+#: views/_ssid.html.php:44
+msgid "Notice"
+msgstr "Notice"
+
+#: views/layout.html.php:66
+msgid "Any problem? Contribute!"
+msgstr "Un souci ? Contribuez !"
+
+#: views/settings.html.php:37
+msgid "Wifi Hotspot Configuration"
+msgstr "Configuration du point d'accès"
+
+#: views/settings.html.php:39 views/settings.html.php:41
+msgid ""
+"This is a fast status. Click on More details to show the complete status."
+msgstr "Ceci est un résumé du statut. Cliquez sur Plus de détails pour consulter l'intégralité du statut."
+
+#: views/settings.html.php:39
+msgid "Running"
+msgstr "En cours d'exécution"
+
+#: views/settings.html.php:41
+msgid "Not Running"
+msgstr "Etient"
+
+#: views/settings.html.php:44
+msgid "Loading complete status may take a few minutes. Be patient."
+msgstr "Le chargement du statut peut prendre quelques minutes. Soyez patient."
+
+#: views/settings.html.php:44
+msgid "More details"
+msgstr "Plus de détails"
+
+#: views/settings.html.php:60
+msgid "Service"
+msgstr "Service"
+
+#: views/settings.html.php:65
+msgid "Hotspot Enabled"
+msgstr "Point d'accès activé"
+
+#: views/settings.html.php:73
+msgid "Device"
+msgstr "Appareil"
+
+#: views/settings.html.php:85
+msgid "Channel"
+msgstr "Cannal"
+
+#: views/settings.html.php:116
+msgid "Add a hotspot"
+msgstr "Ajouter un point d'accès"
+
+#: views/settings.html.php:122
+msgid ""
+"You are currently connected through the wifi hotspot. Please, confirm the "
+"reloading, wait for the wifi disconnect/reconnect and go back here to check "
+"that everything is okay."
+msgstr "Vous êtes actuellement connecté(e) au point d'accès WiFi. Veillez confirmer le rechargement, attendre que le WiFi se déconnecte/reconnecte et revenir ici pour vérifier que tout est correct."
+
+#: views/settings.html.php:124 views/settings.html.php:130
+msgid "Reloading may take a few minutes. Be patient."
+msgstr "Le rechargement peut prendre quelques minutes. Soyez patient."
+
+#: views/settings.html.php:124
+msgid "Confirm"
+msgstr "Confirmer"
+
+#: views/settings.html.php:128 views/settings.html.php:130
+msgid "Save and reload"
+msgstr "Sauvegarder et recharger"
+
+#: views/_ssid.html.php:3
+msgid "Hotspot"
+msgstr "Point d'accès"
+
+#: views/_ssid.html.php:7
+msgid "Wifi"
+msgstr "WiFi"
+
+#: views/_ssid.html.php:8
+msgid "IPv6"
+msgstr "IPv6"
+
+#: views/_ssid.html.php:9
+msgid "IPv4"
+msgstr "IPv4"
+
+#: views/_ssid.html.php:15
+msgid "Name (SSID)"
+msgstr "Nom (SSID)"
+
+#: views/_ssid.html.php:22
+msgid "Secure"
+msgstr "Sûr"
```

Comment:
Hum, we can maybe traduce with something like « sécurisé » no? (the checkbox goal is to enable or not the open hotspot).